Automattic/wp-calypso
# Add Global Site View sidebar

Related to 
- https://github.com/Automattic/dotcom-forge/issues/5550
- https://github.com/Automattic/wp-calypso/pull/87454

## Proposed Changes

This PR adds the new Global Site View sidebar behind the feature flag `layout/dotcom-nav-redesign`. I left the comments for follow-up tasks. 

<img width="275" alt="Screenshot 2024-02-15 at 16 12 42" src="https://github.com/Automattic/wp-calypso/assets/5287479/45bfb919-6081-4881-85ca-e99d23f6174a">

See pe7F0s-1IA-p2 for more context.

## Testing Instructions



* Open Calypso Live
* Prepare an Atomic site with the classic view
* Go to any site-specific Calypso page URL (e.g., /home/<your-site>, plans/<your-site>) 
* See the sidebar works

## Pre-merge Checklist
